build.sbt

```diff
@@ -76,16 +76,31 @@ lazy val publishSettings : Seq[Setting[_]] = Seq(
 // should not be set directly. It is the same as the Maven version and derived automatically from `baseVersion` and
 // `baseVersionSuffix`.
 globalVersionSettings
-baseVersion in Global := "2.13.0"
-baseVersionSuffix in Global := "SNAPSHOT"
+Global / baseVersion       := "2.13.0"
+Global / baseVersionSuffix := "SNAPSHOT"
+ThisBuild / organization   := "org.scala-lang"
+ThisBuild / homepage       := Some(url("https://www.scala-lang.org"))
+ThisBuild / startYear      := Some(2002)
+ThisBuild / licenses       += (("Apache-2.0", url("https://github.com/scala/scala/blob/2.13.x/LICENSE")))
```

Comment:
Is it worth maintaining this URL every major version, or using, perhaps, https://www.apache.org/licenses/LICENSE-2.0 ?

Reply:
We should also update our pom generation, and link to apache.org instead of scala-lang.org